content/blog/2013/susy-next.md

```diff
@@ -52,7 +52,7 @@ name.
 Please welcome the Susy Next team:
 
 - [Jina Bolton][] \[[@jina]\] is a Product Designer at Do, a regular
-  speaker at web conferences, co-creator of the [320 and Up]
+  speaker at web conferences, co-creator of the 320 and Up
```

Comment:
@mirisuzanne A lot of these bios are now out-of-date, but maybe better to leave that in the context of when it was written?